site/external.yaml

```diff
@@ -1 +1 @@
-FybrikRelease:  __Release__
+FybrikRelease:  v1.1.1

```

Comment:
Why 1.1.1?

Reply:
Leftover from testing another PR. Thanks for the catch!